traffic_ops/traffic_ops_golang/api/change_log.go

```diff
@@ -37,6 +37,10 @@ type ChangeLog struct {
 	LastUpdated tc.Time `json:"lastUpdated" db:"last_updated"`
 }
 
+type Logger interface {
+	ChangeLogMessage(action string, db *sqlx.DB) (string, error)
+}
+
```

Comment:
since it's the ChangeLog, what about

type ChangeLog Interface{
    Message{}
}

Reply:
I agree that Logger should be changed but should conform to the -er pattern.